### PR TITLE
Add CLI to configure YANG config validation

### PR DESCRIPTION
**Pull request by isabelmsft**
**- What I did**
Add CLI to configure YANG config validation mode
`config yang_config_validation <enable|disable>`

**- How I did it**
Add a CLI script that writes the configuration of YANG config validation enable/disable into CONFIG_DB

**- How to verify it**

**- Previous command output (if the output of a command-line utility has changed)**
N/A

**- New command output (if the output of a command-line utility has changed)**

### CODE REVIEW

File: config/yang_validation_service.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            loaded_models_sy = sonic_yang.SonicYang(self.yang_dir)
+            loaded_models_sy.loadYangModel()
+            self.sonic_yang_with_loaded_models = loaded_models_sy
+
+        return copy.copy(self.sonic_yang_with_loaded_models)
+    
+    def validate_config_db_config(self, config_json):
+        sy = self.create_sonic_yang_with_loaded_models()
+        try:
+            tmp_config_json = copy.deepcopy(config_json)
```
Comment (wen587) [inline config/yang_validation_service.py:33]: Is the deepcopy necessary?

File: config/main.py [hunk 2]
```diff
@@ -1754,17 +1766,18 @@ def add_portchannel(ctx, portchannel_name, min_links, fallback):
         fvs['min_links'] = str(min_links)
     if fallback != 'false':
         fvs['fallback'] = 'true'
-    db.set_entry('PORTCHANNEL', portchannel_name, fvs)
+    yvs = YangValidationService()
+    if not yvs.validate_set_entry('PORTCHANNEL', portchannel_name, fvs):
+        ctx.fail("Invalid configuration based on PortChannel YANG model")
+    else:
+        db.set_entry('PORTCHANNEL', portchannel_name, fvs)
+
```
Comment (wen587) [inline config/main.py:1774]: It seems we always validate yang no matter `yang_config_validation` is enabled or not?
Maybe we are supposed to validate it based on the `yang_config_validation`?

File: config/main.py [hunk 3]
```diff
@@ -1714,6 +1715,21 @@ def synchronous_mode(sync_mode):
     else:
         raise click.BadParameter("Error: Invalid argument %s, expect either enable or disable" % sync_mode)
 
+#
+# 'yang_config_validation' command ('config yang_config_validation ...')
+# 
+@config.command('yang_config_validation')
+@click.argument('yang_config_validation', metavar='<enable|disable>', required=True)
+def yang_config_validation(yang_config_validation):
```
Comment (qiluo-msft) [inline config/main.py:1723]: >yang_config_validation [](http://example.com/codeflow?start=4&length=22)

Let's focus on `config yang_config_validation` in this PR, and move the sample implementation of yang validation to another PR.

**Comment by wen587** [comment]
LGTM

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**